test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -479,6 +479,7 @@ TEST_F(ClusterManagerImplTest, PrimaryClusters) {
     type: eds
     eds_cluster_config:
       eds_config:
+        resource_api_version: V3
```

Comment:
Question for @htuch: do you think we can ever get away from having to specify V3 all over the place? It complicates the configuration for little benefit now that we're all V3. Would it be possible to switch AUTO to V3?

Reply:
yeah I was really surprised AUTO was v2 which I hadn't realized until I debugged this test breakage  I'd started switching auto to v3 in switch statements and decided it was probably a separate PR if I was going to do that and just updated the test instead.

Reply:
During migration that was not done because it would have caused working configurations to break when folks wanted to do transitional rollouts. I think we could, at this point, consider the AUTO mode to be a client-specific setting, so Envoy can unilaterally decide to interpret it as V3.

Reply:
I'm happy to do that as a follow-up - I don't think it makes a difference with this PR since we're ignoring the version entirely.  
Are there any changes you think this PR needs?